### DISCUSSION TITLE
How can I change the UI Font?

### DISCUSSION THREAD

**Discussion by ghostgabriel** [Godiesc/firefox-gx#211] [Q&A] [answered]
Hello, the theme is great. I loved the tab shape. It is possible to change the ui font? i tried putting this code but it resets the theme.

**the css code:**
{
   font-family: "sf pro display" !important;
}

Thanks!

**Comment by Godiesc** [comment, marked as answer]
hi,
I'm glad you like it, and I was never change the font but with a quick test the following symbol is missing * , you can use the `chrome/components/ogx_tricks.css` file to not break anything, that is the file for your personal code:

```
*{
font-family: "sf pro display" !important;
}
```

**Reply by ghostgabriel** [reply]
Thanks. It worked now.
